tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Datadog.Trace.ClrProfiler.IntegrationTests.csproj

```diff
@@ -21,12 +21,15 @@
 
     <ProjectReference Include="..\..\src\Datadog.Trace\Datadog.Trace.csproj" />
     <ProjectReference Include="..\Datadog.Trace.TestHelpers\Datadog.Trace.TestHelpers.csproj" />
+    <ProjectReference Include="..\test-applications\debugger\Samples.Probes\Samples.Probes.csproj" />
```

Comment:
The test helper infrastructure for running samples does not require actually referencing the project (though in the VS solution you could add a build dependency). Can we remove this or do you need to compile against it?

Reply:
Nevermind, I see several uses where you reference types from the sample. Looks like this is needed 👍🏼

Reply:
Yeps, this is actually something we really didn't like doing as it may could come back to haunt us in a myriad of ways, as we've discussed extensively with @andrewlock. 

There are two potential mitigations we thought of:

1. Remove the direct reference and come up with creative solutions for the use cases we're currently using it for (using duck typing with an interface, using a special syntax of comments to denote where the  probes should be placed and somehow parsing those, etc etc). 
2. Keep the direct reference but move all of Live Debugger's integration tests into a separate .csproj, so we'll still have the added CI complexity that the direct reference causes but at least it will be a bit more isolated.

For now, we've decided to keep it, as we can live with it, but it's definitely something we want to reconsider in the future.